README.rst

```diff
@@ -615,6 +615,10 @@ Choose from the list of available fixers:
                         construct to print if
                         possible.
 
+* **empty_phpdoc** [contrib]
```

Comment:
No need to ask at first, I could send the PR (in the and you can treat PR as extra type of issue with code attached).
But seriously, I don't think there will be any violation for that rule.

The only case I met and why I want that fixer is our annotation cleaning fixers - eg when we remove `@access` annotation and there is nothing left in docblock